modules/rubiconBidAdapter.js

```diff
@@ -716,6 +720,16 @@ export const spec = {
     }, []).sort((adA, adB) => {
       return (adB.cpm || 0.0) - (adA.cpm || 0.0);
     });
+
+    let fledgeAuctionConfigs = responseObj.component_auction_config?.map(config => {
+      return { config, bidId: config.bidId }
+    });
+
+    if (fledgeAuctionConfigs) {
```

Comment:
`return {bids, fledgeAuctionConfigs}` should work, core will ignore the second property if it's not an array and still pick up the bids

Reply:
While that would be better, it breaks some unit tests it's not worth fixing right now. I'll plan to deal with this when we aren't trying to get this out so quickly